wallet/chain/p/context.go

```diff
@@ -21,42 +21,40 @@ const gasPriceMultiplier = 2
 
 func NewContextFromURI(ctx context.Context, uri string) (*builder.Context, error) {
 	infoClient := info.NewClient(uri)
-	xChainClient := avm.NewClient(uri, "X")
```

Comment:
In order to support partial sync, the P-chain wallet can not depend on any X-chain APIs.